shell/platform/darwin/ios/BUILD.gn

```diff
@@ -373,6 +373,12 @@ copy("copy_framework_module_map") {
   outputs = [ "$_flutter_framework_dir/Modules/module.modulemap" ]
 }
 
+copy("copy_framework_privacy_manifest") {
```

Comment:
Maybe a comment linking to https://developer.apple.com/documentation/bundleresources/privacy_manifest_files here to provide context on what this is and why we are doing it?

Although developer.apple.com links are notoriously fragile, so 🤷🏻

Reply:
We don't explain Info.plists, or modulemaps, or the values within those files, or any other thing we move around in this gn file.  In 2 years when you Google "PrivacyInfo.xcprivacy" you'll see the most relevant info, and when you `git blame` you'll find this PR with my thorough description.  However adding the link is harmless so I'll do that.